projects/core/src/cart/connectors/delivery/cart-delivery.adapter.ts

```diff
@@ -0,0 +1,62 @@
+import { Observable } from 'rxjs';
+import { Address, DeliveryMode } from '../../../occ/occ-models/occ.models';
+
+export abstract class CartDeliveryAdapter {
```

Comment:
Should the address related methods not be part of an address service? Or is this specific for the cart API?

Reply:
It looks like it is specific for a cart, to be more precise: `users/user_id/carts/cart_id/addresses/delivery`